ambari-web/app/views/main/service/info/summary/hdfs/widgets.js

```diff
@@ -77,7 +77,7 @@ App.HDFSSummaryWidgetsView = Em.View.extend(App.NameNodeWidgetMixin, App.HDFSSum
 
   dfsTotalFilesValue: Em.computed.getByKey('model.dfsTotalFilesValues', 'hostName'),
 
-  dfsTotalFiles: Em.computed.formatUnavailable('model.dfsTotalFilesValue'),
+  dfsTotalFiles: Em.computed.formatUnavailable('dfsTotalFilesValue'),
```

Comment:
This change would break the functionality because `dfsTotalFilesValue` is taken from `model`, not from the current view object

Reply:
@aBabiichuk   i didnt see the model holding a value  :  dfsTotalFilesValue

also i see here : https://github.com/apache/ambari/blob/7c68f8d4b9ef32dcd8666786bfbf51ad79b68859/ambari-web/app/views/main/service/info/summary/hdfs/widgets.js#L72 that other variables dfsMissingBlocks are taking value from this current View object only. i followed the trend. 

it works in my cluster correctly and data in shown in widget. can you please see if model actually contains this variable to be referenced here.

Reply:
Sorry, I was confused by single `s` letter. `dfsTotalFilesValues` is defined in `model`, and then mapped to `dfsTotalFilesValue` in the view, so you're correct